checkstyle.xml

```diff
@@ -342,11 +342,8 @@
 		<module name="JavadocMethod">
 			<property name="scope" value="public"/>
 			<property name="allowMissingParamTags" value="true"/>
-			<property name="allowMissingThrowsTags" value="true"/>
```

Comment:
Were these properties just removed, or moved to new places as `LineLength` above?

Reply:
These properties are no-longer being supported by `JavadocMethod` so their presence makes the build fail.

Reply:
What were they replaced with?

Reply:
Am taking more investigation on this!